src/kaleidoscope/plugin/NumPad.h

```diff
@@ -16,7 +16,12 @@
 
 #pragma once
 
-#include "Kaleidoscope-LEDControl.h"
+#include <stdint.h>                             // for uint8_t
+#include "kaleidoscope/KeyAddr.h"               // for KeyAddr
+#include "kaleidoscope/event_handler_result.h"  // for EventHandlerResult
+#include "kaleidoscope/plugin.h"                // for Plugin
+
+struct cRGB;
```

Comment:
Code change rather than header change here.

Reply:
Where?

Reply:
Ah, you probably mean the forward declaration? 
The distinction between code change and header change makes no sense IMHO. Adding and removing headers changes the code.